src/gl/metal/vcGLState.mm

```diff
@@ -113,9 +113,8 @@ bool vcGLState_Init(SDL_Window *pWindow, vcFramebuffer **ppDefaultFramebuffer)
   _renderer = [vcRenderer alloc];
   [_renderer initWithView:sdlview];
 
-  vcTexture *defaultTexture, *defaultDepth;
+  vcTexture *defaultTexture, *defaultDepth = nullptr;
```

Comment:
This fix won't set `defaultTexture` to NULL

Reply:
It didn't need to, `defaultDepth` is no longer set.

Reply:
In hindsight, I'm not sure if I needed to remove the depth texture in the end. I vaguely recall there being issues with it - lots of work is required for Metal, so I might loop back to this anyway.

Reply:
Turns out this change wasn't necessary.

Reply:
I fixed this up.